test_e2e/python/test-ingest-delta-table-output.py

```diff
@@ -4,11 +4,20 @@
 
```

Comment:
Initially this .py file was used by `test_e2e/dest/delta-table.sh` and when I added my test (`test_e2e/dest/delta-table-s3.sh`) I modified this .py file to support connection to s3 too. If this file gets removed then `test_e2e/dest/delta-table.sh` should be removed too. Should I do it, or maybe it would be better to undo the changes that I did there and test local Delta Tables with  `test_e2e/dest/delta-table.sh`  and this .py file (as it was initially implemented), and testing uploading Delta Tables to s3 in integration test?